apps/src/p5lab/gamelab/Exporter.js

```diff
@@ -71,55 +53,14 @@ export default {
       animationListJSON
     });
 
-    const iconPath = '/appassets/icon.png';
-    const splashImagePath = '/appassets/splash.png';
-
-    if (expoMode) {
-      appAssets.push({
-        url: exportExpoWarningPng,
-        zipPath: appName + '/appassets/warning.png',
-        dataType: 'binary'
-      });
-      appAssets.push({
-        url: exportExpoIconPng,
-        zipPath: appName + iconPath,
-        dataType: 'binary'
-      });
-      appAssets.push({
-        url: exportExpoSplashPng,
-        zipPath: appName + splashImagePath,
-        dataType: 'binary'
-      });
-    }
-
-    const mainProjectFilesPrefix = appName + (expoMode ? '/assets/' : '/');
+    const mainProjectFilesPrefix = appName + '/';
 
     var zip = new JSZip();
-    if (expoMode) {
-      const appJson = exportExpoAppJsonEjs({
-        appName,
-        sdkVersion: EXPO_SDK_VERSION,
-        projectId: project.getCurrentId(),
-        iconPath: '.' + iconPath,
-        splashImagePath: '.' + splashImagePath
-      });
-      const appJs = exportExpoAppEjs({
-        appHeight,
-        appWidth,
-        hasDataAPIs: false
-      });
-
-      zip.file(appName + '/package.json', exportExpoPackageJson);
-      zip.file(appName + '/app.json', appJson);
-      zip.file(appName + '/App.js', appJs);
-      zip.file(appName + '/CustomAsset.js', exportExpoCustomAssetJs);
-      zip.file(appName + '/DataWarning.js', exportExpoDataWarningJs);
-      zip.file(appName + '/metro.config.js', exportExpoMetroConfigJs);
-    }
+    // remove/update comment
```

Comment:
Same comment here as the other `remove` comment. Intended to leave? And if so, can we add more context?

Reply:
Did not intend to leave, thanks for looking through this in detail! [Found a version of this file from way back](https://github.com/code-dot-org/code-dot-org/blame/5244615f60b6997372ff05f0bf644e621b280677/apps/src/applab/Exporter.js), just before the ["expo needs this to come first" was added](https://github.com/code-dot-org/code-dot-org/blame/06553fabba7ea9d659de1229655afc2d6c2cfd52/apps/src/applab/Exporter.js). Looks like index.html already came first, but for some reason was important to expo to remain that way. Deleted the comment, as well as the one in the Applab exporter.